### PR TITLE
Bump dugite-native for git-2.13.3

### PR DESCRIPTION
**Pull request by joshaber**
@shiftkey Am I doing this right?

### CODE REVIEW

**Comment by joshaber** [comment]
It would seem I am _not_ doing this right.

File: .vscode/settings.json [hunk 1]
```diff
@@ -4,5 +4,6 @@
     "**/.git": true,
     "**/.DS_Store": true,
     "build": true
-  }
+  },
+  "editor.formatOnSave": false
```
Comment (joshaber) [inline .vscode/settings.json]: I added this to avoid Prettier trying to prettify everything I did 🤷‍♂️

File: test/fast/lfs-test.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ describe('lfs', () => {
     const testRepoPath = temp.mkdirSync('desktop-git-lfs')
     const result = await GitProcess.exec([ 'lfs' ], testRepoPath)
     expect(result.exitCode).to.equal(0)
-    expect(result.stderr).to.contain('Git LFS is a system for managing and versioning large files')
+    expect(result.stdout).to.contain('Git LFS is a system for managing and versioning large files')
```
Comment (joshaber) [inline test/fast/lfs-test.ts]: I'm not totally sure why this changed 🤷‍♂️

Comment (shiftkey) [inline test/fast/lfs-test.ts]: I think this is something that was actually fixed in Git LFS: https://github.com/git-lfs/git-lfs/pull/2210

**Comment by shiftkey** [comment]
`1.37.0` has been published to NPM :tada:

**Pull request merged**